test/odbc/src/drivers.cpp

```diff
@@ -0,0 +1,95 @@
+#include "drivers.h"
+#include <fstream>
+
+using std::pair;
+
+Drivers::Drivers() {
+  if (odbc_drivers_.empty()) {
+    SetDrivers();
+  }
+}
+
+Drivers::~Drivers() {
+}
+
+void Drivers::SetDrivers() {
+
+  map<string, string> config_file_values = ParseConfigFile();
+
```

Comment:
We should just return here if config_file_values is empty.
Perhaps when someone will be adding things to this framework or just adding test, they could make a change here.

Reply:
Won't enter the loop if the config file is empty. I don't believe this change would be needed.